apps/vscode/command_client/neovim.py

```diff
@@ -0,0 +1,18 @@
+from talon import Context, actions
+
+ctx = Context()
+
+ctx.matches = r"""
+app: neovim
```

Comment:
Is this app already defined somewhere that I missed?

Reply:
It is only defined in neovim-talon atm.